### PR TITLE
e2e,compute: Add sig-compute-serial e2e target

### PR DESCRIPTION
**Pull request by brianmcarey**
### What this PR does

These new targets allow the splitting of the sig-compute presubmit lane
into a lane with the parallel tests and a separate lane with the serial
tests.

The sig-compute lane is by far the longest running lane - over 3.5 hours
on average[1]. Splitting this lane will give quicker CI feedback to contributors

[1] https://grafana.ci.kubevirt.io/d/gY4hNc5Iz/kubevirt-kubevirt-e2e-test-runtimes?orgId=1&viewPanel=2&from=now-90d&to=now

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 
Mailing list: https://groups.google.com/g/kubevirt-dev/c/YhD-XNUeJJc

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

**Review by dhiller — COMMENTED** [review]
/approve

/hold for others to chime in

File: automation/test.sh [hunk 1]
```diff
@@ -423,6 +427,13 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} && -z ${label_filter}
     label_filter='(sig-compute-realtime)'
   elif [[ $TARGET =~ sig-compute-migrations ]]; then
     label_filter='(sig-compute-migrations && !(GPU,VGPU))'
+  elif [[ $TARGET =~ sig-compute-serial ]]; then
+    export KUBEVIRT_E2E_PARALLEL=false
+    ginko_params="${ginko_params} --focus=\\[Serial\\]"
+    label_filter='((sig-compute) && !(GPU,VGPU,sig-compute-migrations))'
```
Comment (xpivarc) [inline automation/test.sh:433]: maybe we should have a variable that can be reused for the 3 cases.

**Review by xpivarc — APPROVED** [review]

**Comment by brianmcarey** [comment]
/override pull-kubevirt-fossa

fossa is failing on the main branch with the same issue[1] - not related to this PR.

[1] https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/push-kubevirt-fossa/1785802418752065536

**Pull request merged**